README.md

```diff
@@ -200,6 +200,22 @@ require("oil").setup({
     padding = 2,
     max_width = 0,
     max_height = 0,
+    -- Positioning the floating window at (row, col) relative to "editor".
+    -- For more details, see `nvim_open_win()`
+    -- row and col can optionally be a function or an integer.
+    -- When (row, col) are not provided, the window is centered automatically.
+    -- row = 0 means place the floating window at row = 0
+    row = 0,
+    -- col = <func> means place the floating window at the return value of <func>
+    col = function()
+      return vim.o.columns
+    end,
+    -- anchor can optionally be a string. anchor positions the floating window.
+    -- Can be "NE", "NW", "SE", or "SW". By default, "NW" is used.
+    anchor = "NE",
+    -- z-index can optionally be an integer, by default it is 45.
+    -- Higher the z-index, greater is the stacking order of the floating window.
+    zindex = 80,
```

Comment:
This part of the readme is actually autogenerated from the config.lua file, so the changes have to be made there instead.

Reply:
I made the necessary changes in config.lua, instead of README.md. Regarding the override function, I wrote up something similar at first too. The problem I faced was that ```nvim_open_win``` takes up integers or fractions. I deliberately wanted them to take a function as argument, as I wanted the oil window to move right when I resize the editor. I might take a look into nvim-notify, as their floating window seems to move to the right as soon as we resize NeoVim. Do you think that would be a good idea?